### ISSUE TITLE
Evaluation on the cityscapes ( label-> photo) task on the Cityscapes validation set

### ISSUE THREAD

**Issue by lkiani** [junyanz/pytorch-CycleGAN-and-pix2pix#647]
Which images should I use in the evaluation code?
I compared the generated images with Ground truth images and generated images with labels but got an error and the result of all the parameters was zero.
Generated images should be converted to label?
Solving this problem is very important to me.
thank you
![57882367-c4658500-7838-11e9-9e31-695852fd153d](https://user-images.githubusercontent.com/50620477/57967730-a05c8d80-7977-11e9-8b67-d2e6b3a3ece9.png)
![57882413-e101bd00-7838-11e9-8ffe-b0ef87f6e346](https://user-images.githubusercontent.com/50620477/57967731-a05c8d80-7977-11e9-9dd5-d564c96154ac.png)

**Comment by junyanz** [comment]
Please discuss it in this [issue](https://github.com/phillipi/pix2pix/issues/174). I will close this one.
